src/status_im2/contexts/quo_preview/main.cljs

```diff
@@ -367,7 +368,9 @@
                        {:name      :token-input
                         :component token-input/preview}
                        {:name      :wallet-overview
-                        :component wallet-overview/preview-wallet-overview}]
+                        :component wallet-overview/preview-wallet-overview}
+                       {:name      :wallet-activity
```

Comment:
can we keep this alpha sorted, i.e wallet-activity should be before wallet-overview, right?